src/containers/myHouse/addition/ui.tsx

```
@@ -62,37 +63,37 @@ const Addition = () => {
 	const [selectedFacilities, setSelectedFacilities] = useState<string[]>([]);
-	
+
 	const handleFacilityChange = (facility: string, checked: boolean) => {
 		setSelectedFacilities((prev) =>
 			checked ? [...prev, facility] : prev.filter((f) => f !== facility)
 		);
 	};
-	
-	const {error, success} = useAlertStore();
+
+	const { error, success } = useAlertStore();
```

ESLint

[error] 'success' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)